tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -486,6 +489,130 @@ def validate_executor_placements(cls, executor_placements):
                   'Unsupported cardinality for placement "{}": {}.'.format(
                       pl, pl_cardinality))
 
+  async def _trusted_aggregator_generate_keys(self):
+    
+    @computations.tf_computation()
+    def generate_keys():
+      pk, sk = easy_box.gen_keypair()
+      return pk.raw, sk.raw
+
+    fn_type = generate_keys.type_signature
+    fn = generate_keys._computation_proto
+
+    aggregator_ex = self._get_child_executors(placement_literals.AGGREGATOR)[0]
```

Comment:
fyi, you can pass index=0 as a kwarg instead of indexing the result

Reply:
ah good point, thanks

Reply:
please resolve when resolved, makes it easier to follow what remains! 👍